setup.py

```diff
@@ -16,7 +16,7 @@
     packages=find_packages(exclude=('tests.*', 'tests', 'example')),
     install_requires=[
         'Django>=1.8',
-        'sqlparse',
+        'sqlparse>=0.2.0',
```

Comment:
I'd add upper limit as well:
```python
        'sqlparse >=0.2.0,<0.3',
```
This will help to avoid similar issues with unexpected API changes.